src/index.html

```diff
@@ -9,5 +9,56 @@
   </head>
   <body>
     <h1>Stars</h1>
+    <div class="page">
+      <div class="page__container">
+        <div class="stars stars--0">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="stars stars--1">
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="stars stars--2">
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
```

Comment:
Need to fix it everywhere
```suggestion
          <div class="stars__star"></div>
          <div class="stars__star"></div>
```